app/models/vote.rb

```diff
@@ -0,0 +1,11 @@
+class Vote < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :review
+
+  validates :vote_value,
+            inclusion: { in: [1, -1] },
+            presence: true#,
+            # uniqueness: {scope: :review, message: ": You can't upvote twice"}
+  validates :user, presence: true
+  validates :review, presence: true
```

Comment:
The `{ scope: :review }` should validate the uniqueness of the `user-review_id` combination: maybe something wasn't setup correctly? You also have to ensure that the unique index matches this validation as well:

```ruby
add_index :votes, [:review_id, :user_id], unique: true
```